pgcli/packages/sqlcompletion.py

```diff
@@ -20,20 +20,27 @@
 Special = namedtuple('Special', [])
 Database = namedtuple('Database', [])
 Schema = namedtuple('Schema', [])
+# AliasableTable is used in those cases we might want to generate an alias
+AliasableTable = namedtuple('AliasableTable', 'schema tables')
```

Comment:
Could we also add a comment explaining that the tables field is accomplishing here? If I understand correctly, correctly suggesting a table & alias requires knowing what aliases have already been used